server/src/lib.rs

```diff
@@ -54,6 +48,6 @@ pub struct Cli {
 pub async fn shutdown_signal(server: &str) {
     tokio::signal::ctrl_c()
         .await
-        .expect("expect tokio signal ctrl-c");
-    rest_warn!("({}) shutdown signal", server);
+        .expect("(shutdown_signal) expect tokio signal ctrl-c.");
+    rest_warn!("({}) shutdown signal.", server);
```

Comment:
should be prefixed with function name:
`log::warn!("(shutdown_signal) server shutdown for [{}].", server);`

Reply:
https://github.com/Arrow-air/svc-atc/pull/3/commits/48b57f0303a3111a30e515a7edd4c2514d95178f